### DISCUSSION TITLE
What happens next?  Understanding court record (no identifying information shared)

### DISCUSSION THREAD

**Discussion by jcinct** [freelawproject/courtlistener#3339] [General]
A federal case I am following had the following entry on October 11th.  I do not see a future action noted so I am trying to determine if the case is closed or if there is a future court date.  I am unsure if I am allowed to share details so I will keep it to...the case is in regards to statements made threatening to carry out a school shooting that resulted in state and federal charges. 

"ORDER: Canceling the probable cause hearing scheduled for October 13, 2023 at 9:00 a.m. before Judge XXX as to XXX, re 19 Calendar Entry. Signed by Judge XXX on 10/11/23. (Entered: 10/11/2023)"

A psychiatric exam was also ordered that day:

"ORDER granting 20 Motion for Psychiatric Exam as to XXX. Signed by Judge XXX on 10/11/23. (Entered: 10/11/2023)"

This is the end of the record. I cannot see anything referring to future dates. Wondering if the case is dropped or if it is still continuing. Parents at the school in question would like to know status but it is very hard to find. 

Thank you.

**Comment by johnhawkinson** [comment]
This isn't really a question about Courtlistener, and we're not equipped to address questions about how the federal courts work.

That said, this situation is pretty clear:

> A case I am following had the following entry on October 11th. I do not see a future action noted so I am trying to determine if the case is closed or if there is a future court date.

No and No. it is not closed, and there is no future court date.

It is pretty easy to look at the case in PACER, and for free you can see that it does not say "Case Closed" or "Case Terminated" at the top, and it does say "Date of last filing: 10/11/2023." That query is even free.

It's pretty clear here that both parties jointly asked for a psychiatric exam, and the judge granted the motion (as almost always happens when both parties jointly agree to ask for something), and consequently cancelled the hearing he had previously scheduled, almost certainly because the hearing would not make sense without the results of the psychiatric exam, which would not happen within two days. There may or may not be communications about this stuff that appropriately do not show up on the docket, and I have not reviewed the filings here, but this all makes sense.

Presumably the parties will report the results of the psychiatric exam to the Court, possibly under seal, and after that happens something will happen in the case, whether scheduling a hearing or disposing of the case or something else.

**Reply by jcinct** [reply]
Is there a way to see when the exam has to be completed? When I click to buy document on PACER it says document is restricted or cannot be found so I am unable to determine when exam is supposed to happen by.  I am figuring the prosecution would want dates set so they can move along and determine future action.

**Comment by johnhawkinson** [comment]
> Is there a way to see when the exam has to be completed?

Almost certainly not. Both because neither the Court nor the Government nor the defense counsel are going to be conducting the examination, and so they lack control over its completion date; and also because that likely may be construed as confidential medical information best conducted outside the public view.

But this doesn't really have anything to do with Courtlistener.
You could ask the parties or the court clerk's office or the probation department how this typically works in this particular district or this particular case, but I suspect you would not get very far.

I suspect the document here is not actually restricted and rather there is no document attached to the docket entry 25. Could be wrong though. Unfortunately Courtlistener does not make this super-clear. There is a document attached to docket entry 24 which might shed some light, though.

That said, there are clearly sealed documents and private medical information in this case, which  justifies the seal (see docket entries 21 and 23), so if you're attempting to read closely, you're going to run up against those limitations sooner or later.

**Reply by jcinct** [reply]
thank you.  Not sure how you know which docket I am referring to but I greatly appreciate your help.  I was able to open line 24 and see that the defendant was committed to the custody of the atty general for placement in a facility and exam timeline parameters were set forth. It goes on to say that once examiners report is back, a competency hearing will be set. 

Much appreciated!  Just hard to follow legal documentation when I am not a lawyer.
